server/mailet/mock-smtp-server/pom.xml

```diff
@@ -113,57 +113,4 @@
             <artifactId>subethasmtp</artifactId>
         </dependency>
     </dependencies>
-
-    <build>
```

Comment:
no explanation about why?

Reply:
![Capture d’écran de 2020-07-31 15-48-29](https://user-images.githubusercontent.com/6928740/89018023-63df3c80-d345-11ea-8d0e-52c15b0b7a51.png)

The full diff does what is claim: require dockerBuild or buildTar to be explicitly specified.

And "why" because JIB execution was costing 2*18 minutes while we don't rely on the build result.

I re-added part of the plugin later, sorry for the confusion.